Makefile

```diff
@@ -0,0 +1,5 @@
+.PHONY: build
+
+build:
+	rm -rf node_modules
+	docker run -it --entrypoint /bin/bash --workdir /indico-sui-theme --mount type=bind,src=`pwd`,target=/indico-sui-theme node:10 build.sh
```

Comment:
what's the benefit of using `--mount ...` instead of `-v $(pwd):/indico-sui-theme`?

Reply:
It is the preferred way of creating volumes now.